lib/v2/screens/explore_screens/vote_screens/vote/interactor/viewmodels/vote_state.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/explore_screens/vote_screens/vote/interactor/viewmodels/current_remaining_time.dart';
+
+/// --- STATE
+class VoteState extends Equatable {
+  final PageState pageState;
+  final int remainingTimeStamp;
+  final CurrentRemainingTime? currentRemainingTime;
```

Comment:
yeah, so this is good nullable, but the variables inside `CurrentRemainingTime` can be not null?

Reply:
It's wrong if I put a const CurrentRemainingTime() in the initial? because in that way i can omit the CurrentRemainingTime? here.